packages/bpk-component-barchart/src/BpkChartAxis.js

```diff
@@ -76,7 +78,25 @@ const getAxisConfig = ({ orientation, margin, height, width, scale }) => {
   };
 };
 
-const BpkChartAxis = props => {
+type Props = {
+  height: number,
+  width: number,
+  margin: {
+    top: number,
+    bottom: number,
+    left: number,
+    right: number,
+  },
+  scale: Function,
+  label: ?Node,
+  orientation: string,
+  tickValue: Function,
```

Comment:
We typically try to avoid `Function` in favour of more specific syntax like `(argType) => returnType`, is there something more specific we can use here?

Reply:
Yeah these caused issues when I tried to type them exactly first time around :/ trying to type it correctly just constantly threw errors so not sure what to change it to here